### PR TITLE
Rh latex fixes

### PR DESCRIPTION
**Pull request by ralphtq**

### CODE REVIEW

File: vocab/systems/VOCAB_QUDT-SYSTEM-OF-UNITS-ALL-v2.1.ttl [hunk 1]
```diff
@@ -63,9 +63,8 @@ sou:SOU_CGS
   qudt:informativeReference "http://www.tf.uni-kiel.de/matwis/amat/mw1_ge/kap_2/basics/b2_1_14.html"^^xsd:anyURI ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/sou> ;
   rdfs:label "CGS System of Units" ;
-  rdfs:seeAlso qudt:SOU_CGS-EMU ;
-  rdfs:seeAlso qudt:SOU_CGS-ESU ;
-  rdfs:seeAlso qudt:SOU_CGS-GAUSS ;
+  rdfs:seeAlso sou:SOU_CGS-EMU ;
+  rdfs:seeAlso sou:SOU_CGS-ESU ;
```
Comment (steveraysteveray) [inline vocab/systems/VOCAB_QUDT-SYSTEM-OF-UNITS-ALL-v2.1.ttl:67]: Why eliminate GAUSS?

File: schema/shacl/SCHEMA_QUDT_NoOWL_v2.1.3.shapes.ttl [hunk 2]
```diff
@@ -1917,7 +1917,7 @@ qudt:SystemOfUnits-qudt_hasDerivedCoherentUnit
 qudt:SystemOfUnits-qudt_hasDerivedUnit
```
[the remark this answers is not in the document]
Comment (ralphtq) [reply schema/shacl/SCHEMA_QUDT_NoOWL_v2.1.3.shapes.ttl:1917]: For whatever reason qudt:DerivedUnit was showing up as undefined - I thought you must have removed the class so I made the range qudt:Unit. Just looked again and qudt:DerivedUnit is there - so I have reversed this change.

File: vocab/systems/VOCAB_QUDT-SYSTEM-OF-UNITS-ALL-v2.1.ttl (hunk 1 above)
Comment (ralphtq) [reply vocab/systems/VOCAB_QUDT-SYSTEM-OF-UNITS-ALL-v2.1.ttl:67]: It was also showing up as undefined. Can you take a look?

Comment (steveraysteveray) [reply vocab/systems/VOCAB_QUDT-SYSTEM-OF-UNITS-ALL-v2.1.ttl:67]: OK, added it back in.

**Pull request merged**